tests/compose

```diff
@@ -49,6 +49,15 @@ mkdir -p ${test_compose_datadir}/cache
 
 echo "Preparing compose tests..." | tee -a ${LOG}
 
+if [ -n "${RPMOSTREE_COMPOSE_TEST_USE_HOST_REPO:-}" ]; then
+  # take the host repo if it matches our target tree
+  (source /etc/os-release;
+   if [ "$ID" == "fedora" ] && [ "$VERSION_ID" == "25" ]; then
```

Comment:
I think this is OK, but...we're going to be in for a bit of pain when we need to go to 26.  Maybe just add the string "fedora-25" here so it's more easily greppable?